pkg/oc/lib/graph/kubegraph/edges.go

```diff
@@ -120,7 +119,7 @@ func AddManagedByControllerPodEdges(g osgraph.MutableUniqueGraph, to graph.Node,
 				continue
 			}
 			if query.Matches(labels.Set(target.Labels)) {
-				g.AddEdge(target, to, ManagedByControllerEdgeKind)
+				g.AddEdge(target, to, appsgraph.ManagedByControllerEdgeKind)
```

Comment:
does not sound right :-) you importing this just to get a const...

Reply:
I had to move that constant, otherwise I was getting import cycles.

Reply:
@soltysh maybe create some common package for const, we can handle that as follow up